### DISCUSSION TITLE
Issue with Chat in VS Code

### DISCUSSION THREAD

**Discussion by fpolli** [TabbyML/tabby#2843] [Q&A] [answered]
Hi,
When I go to the chat tab in VS Code, this is what it says:
```
Please update to the latest release of the Tabby server.

You also need to launch the server with the chat model enabled; for example, use --chat-model Qwen2-1.5B-Instruct.
```
This is my command line (MacBook Pro M3Pro, installed with Homebrew):
`
tabby serve --device metal --model TabbyML/StarCoder-3B --chat-model CodeQwen-7B-Chat
`
This is the splash screen when it starts up:

📄 Version 0.15.0
🚀 Listening at 0.0.0.0:8080

The code completion part works in VS Code.

According to the Code page here, 0.15.0 is the latest release.

Any thoughts?

Thanks.

**Comment by icycodes** [comment, marked as answer]
Hi!  
To troubleshoot this issue, I recommend fetching the server health information from `/v1/health` at http://localhost:8080/api (make sure to set the token first) and comparing it with the [checks listed here](https://github.com/TabbyML/tabby/blob/main/clients/vscode/src/chat/ChatViewProvider.ts#L247).   
Also, if you built the Tabby VSCode extension from source, please note that the code on the main branch requires work with Tabby server version 0.16.0.

**Comment by ms-semarchy** [comment]
Likewise, the chat does not work in VScode (empty tab)

**Reply by ms-semarchy** [reply]
worked after restarting a few times VScode. 
Weird ..
